# This service returns the currently executing task
String yaml
---
Task task
